content/validation/intro.md

```diff
@@ -0,0 +1,32 @@
+# Introduction
+
+In general, ML models don't really work out of the box: for example, mostly often it is not sufficient to simply instantiate the model class, call its `fit()` method followed by `predict()` and proceed straight to the inference step of the analysis:
```

Comment:
"...box. For example, most often it is not sufficient to simply instantiate the model class, call its `fit()` method followed by `predict()`, and then proceed straight to the inference step of the analysis."